Source/CarthageKit/GitHub.swift

```diff
@@ -266,6 +284,14 @@ internal struct GitHubCredentials {
 	}
 }
 
+internal func loadGitHubCredentials(accessToken: GitHubAccessToken?) -> SignalProducer<GitHubCredentials?, CarthageError> {
+	if let accessToken = accessToken {
+		return SignalProducer(value: AccessTokenGitHubCredentials(accessToken: accessToken))
```

Comment:
Can we read `GITHUB_ACCESS_TOKEN` here instead of passing it in? I think it's best to read it from CarthageKit.